src/commands/deploy.rs

```diff
@@ -693,6 +700,9 @@ async fn wait_for_ready(app_base_url: &Url, bindle_version: &str, readiness_time
         if start.elapsed() >= readiness_timeout {
             println!();
             println!("Application deployed, but Spin could not establish readiness");
+            if deployed_to_cloud {
+                println!("The Fermyon Cloud dashboard may have some logs that explain why readiness could not be established")
```

Comment:
Link maybe?

Also maybe phrase as more of a concrete action a la "Check the Fermyon Cloud (LINK) dashboard to see the application status."